packages/platform/atoms/booker/BookerWebWrapper.tsx

```diff
@@ -187,11 +187,18 @@ export const BookerWebWrapper = (props: BookerWebWrapperAtomProps) => {
       onOverlayClickNoCalendar={() => {
         router.push("/apps/categories/calendar");
       }}
-      onClickOverlayContinue={() => {
-        const newUrl = new URL(`${WEBAPP_URL}/login`);
-        newUrl.searchParams.set("callbackUrl", window.location.pathname);
-        newUrl.searchParams.set("overlayCalendar", "true");
-        router.push(newUrl.toString());
+      onClickOverlayContinue={(provider: "calcom" | "google" = "calcom") => {
+        const currentUrl = new URL(`${WEBAPP_URL}/login`);
+        currentUrl.searchParams.set("overlayCalendar", "true");
+        if (provider === "google") {
+          const url = new URL("/getting-started/connected-calendar", window.location.origin);
```

Comment:
This page requires a logged in user so clicking continue via Google just brings us to the login screen.

Reply:
Fixed.